dev/preview/test/load-test.sh

```diff
@@ -52,19 +53,20 @@ y/n: " yn
     esac
 done
 
-log "Creating base branch ${BASE_BRANCH_NAME}"
-git checkout -b "${BASE_BRANCH_NAME}"
-git commit --allow-empty -m "Prepare load test" -m "/werft with-vm=true"
+FULL_BRANCH_NAME="${BASE_BRANCH_NAME}-${BRANCH_SUFFIX}"
+log "Creating base branch ${FULL_BRANCH_NAME}"
+git checkout -b "${FULL_BRANCH_NAME}"
+git commit --allow-empty -m "Prepare load test"
```

Comment:
We don't need to create a commit anymore now that we don't need to add the Werft commands ☺️